Sources/Subs-Calendar.php

```diff
@@ -36,52 +36,38 @@ function getBirthdayRange($low_date, $high_date)
 	if ($smcFunc['db_title'] !== POSTGRE_TITLE)
 	{
 		// Collect all of the birthdays for this month.  I know, it's a painful query.
-		$result = $smcFunc['db_query']('birthday_array', '
+		$result = $smcFunc['db_query']('', '
 			SELECT id_member, real_name, YEAR(birthdate) AS birth_year, birthdate
 			FROM {db_prefix}members
-			WHERE MONTH(birthdate) != {int:no_month}
-				AND DAYOFMONTH(birthdate) != {int:no_day}
-				AND YEAR(birthdate) <= {int:max_year}
+			WHERE birthdate != \'1004-01-01\'
 				AND (
 					DATE_FORMAT(birthdate, {string:year_low}) BETWEEN {date:low_date} AND {date:high_date}' . ($year_low == $year_high ? '' : '
 					OR DATE_FORMAT(birthdate, {string:year_high}) BETWEEN {date:low_date} AND {date:high_date}') . '
 				)
-				AND is_activated = {int:is_activated}',
+				AND is_activated = 1',
```

Comment:
i was believing that smf coding style is,
that static value also should be placed by placeholder?